### ISSUE TITLE
Error on Rails 6.1.0.rc2: undefined method `resolve_column_aliases' for #<ActiveRecord::PredicateBuilder:0x000055e74c4bfb70>

### ISSUE THREAD

**Issue by kaspernj** [CanCanCommunity/cancancan#666]
### Steps to reproduce

Set my Gemfile to this:
```ruby
gem "cancancan", git: "https://github.com/CanCanCommunity/cancancan.git"
gem "rails", "6.1.0.rc2"
```

### Expected behavior

It shouldn't raise an error.

### Actual behavior

This error is being raised:
```
undefined method `resolve_column_aliases' for #<ActiveRecord::PredicateBuilder:0x000055e74c4bfb70>
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_5_adapter.rb:49:in `sanitize_sql_activerecord5'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_5_adapter.rb:38:in `sanitize_sql'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_adapter.rb:115:in `merge_non_empty_conditions'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_adapter.rb:110:in `merge_conditions'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_adapter.rb:49:in `block in extract_multiple_conditions'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_adapter.rb:48:in `each'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_adapter.rb:48:in `inject'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_adapter.rb:48:in `extract_multiple_conditions'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_adapter.rb:43:in `conditions'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_adapters/active_record_adapter.rb:57:in `database_records'
/home/dev/.rvm/gems/ruby-2.6.6/bundler/gems/cancancan-d284e87464a3/lib/cancan/model_additions.rb:24:in `accessible_by'
```

### System configuration
**Rails version**: 6.1.0.rc2

**Ruby version**: 2.6.6

**CanCanCan version** Current master on Github

**Comment by francesco-loreti** [comment]
Rails 6.1 is been released

**Comment by diegosteiner** [comment]
From what I can tell the method ```resolve_column_aliases``` of ```PredicateBuilder``` has been removed by [this commit in rails](https://github.com/rails/rails/commit/c4adb011e8b0a4474020c685f0c23ec05d99ec9c). CanCanCan uses it [here](https://github.com/CanCanCommunity/cancancan/blob/585e5ea54c900c6afd536f143cde962ccdf68607/lib/cancan/model_adapters/active_record_5_adapter.rb#L49). 

It should be possible to skip Lines 49-52 for Rails 6.1 as the new implementation of ```build_from_hash``` does this by default now.

I'm not sure how you want to make the differentiation to the new Rails Version, but I could provide a PR if needed.

**Comment by coorasse** [comment]
closed in cancancan 3.2.0
